### PR TITLE
Apply hack white

### PR DESCRIPTION
**Pull request by agustin18**

### CODE REVIEW

File: customizations/tests.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def test_post(self, mock_upload_file, mock_webhook, mock_token):
                 u'footer_description': [u'asdasd'],
                 u'show_event_sequence': [u'on'],
                 u'create_customization': [u'Create'],
-                u'select_design_template': [u'DESIGN 1'],
+                u'select_design_template': [u'1'],
```
Comment (cristianemoyano) [inline customizations/tests.py]: This field was eliminated, it should not appear in the test, right?

File: filename [hunk 2]
```diff
@@ -0,0 +1 @@
+tickets/template_default.html
```
Comment (cristianemoyano) [inline filename]: What's this file?

File: customizations/tests.py (hunk 1 above)
Comment (agustin18) [reply customizations/tests.py]: This field continues in the form, the difference is that you should send the number of the template for example in this case the test simulates that we choose the first one

File: filename (hunk 2 above)
Comment (agustin18) [reply filename]: I'm not sure i will ask to francisco he create this file for the hackaton. That is one of muy questions when i did this part.

**Pull request merged**